docs/styleguide.config.js

```diff
@@ -1,8 +1,11 @@
 const path = require('path')
 
 module.exports = {
-  components: path.resolve(__dirname, '../react/{Button,Toggle}/*.{js,jsx}'),
+  components: path.resolve(__dirname, '../react/{Spinner,Icon,Button,Toggle}/*.{js,jsx}'),
   webpackConfig: require('./webpack.config.js'),
   serverPort: 6161,
-  styleguideDir: path.resolve(__dirname, '../build/react')
+  styleguideDir: path.resolve(__dirname, '../build/react'),
+  require: [
```

Comment:
Cannot fund any documentation about `require` in webpack.
What is used for?

Reply:
To require additional files : https://react-styleguidist.js.org/docs/configuration.html#require